vmdb/spec/controllers/chargeback_controller_spec.rb

```diff
@@ -82,4 +82,18 @@
       expect(response.status).to eq(200)
     end
   end
+
+  render_views
```

Comment:
Ya, RSpec requires you to do this if you want views to be rendered.  Unfortunately, RSpec-Rails does not render views by default.